src/core/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/side-navigation.css';
+import { CommonProps } from '../utils/props';
+import SvgChevronRight from '@itwin/itwinui-icons-react/cjs/icons/ChevronRight';
+import { IconButton } from '../Buttons';
+
+export type SidebarProps = {
+  /**
+   * Buttons shown in the top portion sidebar.
+   * Recommended to use `SidenavButton`s.
+   */
+  mainItems: React.ReactNode[];
```

Comment:
Need to think about naming...

Reply:
Any ideas? 😀 Cause I couldn't think other name, maybe just `items` but when I look ar other component libs they are using very simple naming for these kind of stuff like `start/top/leftSomething`.

Reply:
Id think about topItems and bottomItems to be honest.. Directly say, where they are.. Cause not sure if those are main and secondary in workflorw...

Reply:
I like `items` and `secondaryItems`.

"secondary" makes sense because it will only include settings icon or something similar